src/Experimental/RAGTools/rag_interface.jl

```diff
@@ -161,6 +161,13 @@ Main abstract type for storing document chunks and their embeddings. It also sto
 """
 abstract type AbstractChunkIndex <: AbstractDocumentIndex end
 
+"""
+    AbstractPTPineconeIndex <: AbstractDocumentIndex
```

Comment:
Because I didn't want to cause confusion with [`PineconeIndex`](https://github.com/tullytim/Pinecone.jl/blob/main/src/Pinecone.jl#L17-L19) from Pinecone.jl.